src/com/cft/contactmerge/AddressMatchSelector.java

```diff
@@ -0,0 +1,52 @@
+package com.cft.contactmerge;
+
+import java.util.ArrayList;
+
+public abstract class AddressMatchSelector {
+    /*******************************************************************************************************************
+     *********************************************** Street Address Only ***********************************************
+     *******************************************************************************************************************/
+    public static AnswerType getStreetAddressMatchResultDefaultComparisons(String stAddress1, String stAddress2){
+        /* ToDo: Fill in code to return match result for street address...
+         *       Normalize stAddress1 and stAddress2 into their own ArrayList for Matching
+         *          Hint: use StreetAddressMatchLogic.setNormalizeStreetAddress(stAddress)
+         *       Use StreetAddressMatchLogic.getStreetAddressMatchResult(stAddress1, stAddress2)
+         *          to get the return of AnswerType.
+         */
+        return AnswerType.no;
+    }
+
+    public static AnswerType getStreetAddressMatchResultUsrSelectedComparisons(
```

Comment:
What is an example where this would be used? As per our previous discussions, I don't think we should build anything for future use cases. If we don't have a case of this in ITK's data we leave it out until we run into a real use case.